CheckingAccount.java

```diff
@@ -0,0 +1,24 @@
+public class CheckingAccount{
+
+    //instance variables
+    double balance;
+
+    //constructors
+    public CheckingAccount()
+    {
+        balance = 100.00;
+    }
+
+    //methods
+    public double getCheckAccount(){
```

Comment:
This should be `getBalance` since it is a getter method for the variable `balance`.